gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -83,6 +83,63 @@ eink_anr_frei_kinder:
     3:
       upper_threshold: 1500
 
+eink_anr_frei_bürgergeld:
```

Comment:
Why do we need new parameters here? Couldn't we just update `eink_anr_frei`?

Reply:
For sure. Eventually we'll need a params file for `bürgergeld`, but for now let's pretend it is just `arbeitsl_geld_2`